TopMovies/Misc/Date+Extensions.swift

```diff
@@ -14,7 +14,10 @@ extension Date {
   
   static func prettyDate(from string: String) -> Date {
     guard
-      let date = DateFormatter.cached(withFormat: "yyyy-mm-dd")
+      let date = DateFormatter
+        .cached(withFormat: "yyyy-mm-dd")
+        .date(from: string) ?? DateFormatter
+        .cached(withFormat: "yyyy-MM-dd'T'hh:mm:ssZ")
```

Comment:
Try to use ISO8601DateFormatter instead :) 
Don't remember, but it might even work with a common format, without time.